plugins/javadoc/src/test/kotlin/javadoc/JavadocTest.kt

```diff
@@ -14,7 +14,7 @@ class JavadocTest : AbstractCoreTest() {
                 pass {
                     sourceRoots = listOf("jvmSrc/")
                     analysisPlatform = "jvm"
-                    targets = listOf("jvm")
+                    // TODO targets = listOf("jvm")
```

Comment:
What is exactly going on here?

Reply:
I was waiting for the fix in 0.11.0 to arrive. This change does not exist anymore after rebasing onto latest dev 👍